services/core/SQLHistorian/README.md

```diff
@@ -7,7 +7,43 @@ inconsistent network connectivity (automatic re-connection to tcp based
 databases). All additions to the historian are batched and wrapped
 within a transaction with commit and rollback functions properly
 implemented. This allows the maximum throughput of data with the most
-protection.
+protection
+
+## Common Configurations
+All SQLHistorians support a minimum of two parameters
```

Comment:
If the second one is optional then why is it a minimum of 2?

Reply:
good point. Will fix